tests/posix/chmod.rs

```diff
@@ -161,23 +153,17 @@ fn test_chmod_many_options() {
     let original_umask = unsafe {
         umask(0)
     };
-    let tests = vec!{
-        TestCase{args: vec!{"-r,a+w", TEST_FILE}, before: 0o100444, after: 0o100222},
-    };
-    run_tests(tests);
+
+    let tests = vec![
+        (0o100444, "-r,a+w", 0o100222),
+    ];
+
+    for t in tests {
+        let (before, mode, expected) = t;
+        run_test(before, mode, expected);
+    }
+
     unsafe {
         umask(original_umask);
     }
 }
-
-#[test]
-#[ignore]
-fn test_chmod_reference_file() {
```

Comment:
Can you add this back?  It should be supported eventually (I just need to figure out how to get it to work with `clap`).

Reply:
It's OK, because I cannot test my rewrite right now. So, let's write test cases then when it's ready.